pydy/codegen/ode_function_generators.py

```diff
@@ -169,8 +169,8 @@ def _deduce_system_type(**kwargs):
         full mass matrix
             M(x, p) * x' = f(x, t, r, p)
         min mass matrix
-            M(q, p) * u' = f(q, u, t, r, p)
-            q' = g(q, u, t)
+            M(x, p) * x'_d = f(x, t, r, p)
+            x'_k = g(x, t)
```

Comment:
If you change the notation it needs to be:

M(x_q, p) x'_u = f(x, r, p, t)
x_q' = g(x, p, t)

Where x = [x_u, x_q]^-1